lib/project_types/script/cli.rb

```diff
@@ -2,7 +2,7 @@
 
 module Script
   class Project < ShopifyCli::ProjectType
-    hidden_project_type
+    hidden_feature
```

Comment:
Should this be `hidden_feature(feature_set: :script)`?

Reply:
I left this hidden so that your team could decide on the naming of your feature set.